src/endpoints.js

```diff
@@ -61,3 +61,5 @@ export const endpoints = {
       DELETE
     */
 };
+
+export default endpoints
```

Comment:
Don't forget your default export kids! 👍  (It was me, I forgot the default export)